deflate.c

```diff
@@ -105,11 +105,27 @@ static PyObject *deflate_gzip_decompress(PyObject *self, PyObject *args) {
     return output;
 }
 
+static PyObject *deflate_crc32(PyObject *self, PyObject *args) {
+    Py_buffer data;
+    unsigned int crc = 0;
+
+    if (!PyArg_ParseTuple(args, "y*|I", &data, &crc)) {
+        return NULL;
+    }
+
+    crc = libdeflate_crc32(crc, data.buf, data.len);
+    PyBuffer_Release(&data);
+
+    return Py_BuildValue("I", crc);
+}
```

Comment:
https://docs.python.org/3/library/zlib.html#zlib.crc32

it would be cool to have precisely the same api as `zlib.crc32` from python stdlib (needs `value=0` as **optional** parameter for the crc32 start value).

that would make using that pretty easy as a slow fallback in case `deflate` / libdeflate is not available.

Reply:
It's already doing that.

Reply:
Ah, right, just looked up the `|` semantics...